firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/Personalization.java

```diff
@@ -17,18 +17,36 @@
 import android.os.Bundle;
 import androidx.annotation.NonNull;
 import com.google.firebase.analytics.connector.AnalyticsConnector;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
 import org.json.JSONObject;
 
 public class Personalization {
   public static final String ANALYTICS_ORIGIN_PERSONALIZATION = "fp";
-  public static final String ANALYTICS_PULL_EVENT = "_fpc";
-  public static final String ARM_KEY = "_fpid";
-  public static final String ARM_VALUE = "_fpct";
-  static final String PERSONALIZATION_ID = "personalizationId";
+
+  // Constants with LOG_KEY suffix are how the corresponding ones without it are identified on
+  // Google Analytics.
+  public static final String ANALYTICS_PULL_EVENT = "personalization_assignment";
```

Comment:
Minor: it's unclear to me what "pull" means here and below. Also, the `ANALYTICS_..._INTERNAL` naming is helpful below, but we only use it in one place. Given we have external and internal events, and Analytics differentiates "event" from "param" field types (per the [events proposal](go/p13n-events)), something like  `ANALYTICS_EXTERNAL_EVENT` would be more self-descriptive here. In general, I'd vote to use  `ANALYTICS_EXTERNAL_` and `ANALYTICS_INTERNAL_` prefixes to consistently differentiate field names for the external and internal events, and `..._EVENT` and `..._PARAM` to differentiate event names from param names using Analytics' nomenclature.

Thinking: the external event name is arbitrary, we just need to document what it is. `personalization_assignment` seems self-descriptive to me 👍